tests/dvslib/dvs_mirror.py

```diff
@@ -66,14 +67,13 @@ def verify_port_mirror_config(self, dvs, ports, direction, session_oid="null"):
                 assert "SAI_PORT_ATTR_EGRESS_MIRROR_SESSION" not in member.keys() or member["SAI_PORT_ATTR_EGRESS_MIRROR_SESSION"] == "0:null"
 
     def verify_session_db(self, dvs, name, asic_table=None, asic=None, state=None, asic_size=None):
+
+        mirror_session_db_poll = PollingConfig(polling_interval=0.01, timeout=120, strict=True)
```

Comment:
I would wait until the build-tools fix merges, but I think this can probably be removed and we can use the default polling interval.

Reply:
@daall reverted this change. Last run was good.